### PR TITLE
Add trino kerberos connector

### PR DESCRIPTION
**Pull request by garvij**
Enhancement
  - Added trino handler (using kerberos authentication, trino supports many connection types https://trino.io/docs/current/security/authentication-types.html ) Perhaps we can add connection type as a parameter and then plugin in the specific trino-kerberos connector.
mindsdb

### CODE REVIEW

**Comment by ZoranPandovski** [comment]
recheckcla

**Comment by mindsdbadmin** [comment]
**CLA Assistant Lite bot:** <br/>Thank you for your submission, we really appreciate it. Like many open-source projects, we ask that you sign our [Contributor License Agreement]() before we can accept your contribution. You can sign the CLA by just posting a Pull Request Comment same as the below format.<br/>
   - - -
   ***I have read the CLA Document and I hereby sign the CLA***
   - - -
   <sub>You can retrigger this bot by commenting **recheck** in this Pull Request</sub>

**Comment by garvij** [comment]
I have read the CLA Document and I hereby sign the CLA

File: requirements.txt [hunk 1]
```diff
@@ -24,3 +24,5 @@ checksumdir >= 1.2.0
 mindsdb-streams == 0.0.5
 duckdb == 0.3.1
 requests >= 2.0.0
+trino~=0.313.0
```
Comment (ZoranPandovski) [inline requirements.txt:27]: Can you please add the dependencies under the `integrations/trino_handler/` directory in a new `requirements.txt` file?

**Review by ZoranPandovski — CHANGES_REQUESTED** [review]

**Review by ZoranPandovski — APPROVED** [review]

**Pull request merged**